site/styles/_mixins.scss

```diff
@@ -23,3 +23,17 @@
     z-index: 8;
     @include gradient-x( rgba($left-gradient-bg-color, 1), transparent, 0%, 100%);
 }
+
+.a-links{
+    color: $body-color;
+    text-decoration: underline;
+    background-color: transparent;
+    font-weight: bold;
+}
+.a-links-hover-focus{
+    &:hover,
```

Comment:
These can be part of a-links... I don't see any point to them being a separate mixin.  Or am I missing something?

Reply:
Some of the a links have a different hover/focus styles and merging the above ones into that would affect them. For ex, the nav a links have different hover combination

Reply:
I think overriding the mixin styles for those elements is nicer than having to include 2 mixins on all the other anchor elements.